tools/SqliteCompiler/DarwinBuilder.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace SqliteCompiler
+{
+	public class DarwinBuilder
+	{
+		public static void Compile(string version)
+		{
+			var objDir=Path.Combine(Directory.GetCurrentDirectory(), "obj/osx");
+
+			try 
+			{
+				Directory.Delete(objDir, true);
+			} catch {}
+
+			Directory.CreateDirectory(objDir);
+
+			new Command
+			{ 
+				Executable = "curl",
+				Args = $"-sSL https://sqlite.org/2016/sqlite-autoconf-{version}.tar.gz -o src.tgz",
```

Comment:
the link should be a const - it is used in quite a few places. Also looks like getting and extracting source could be shared between darwin and linux